certified-connectors/My Acclaro/intro.md

```diff
@@ -0,0 +1,173 @@
+The Acclaro Web Service API is an extensible set of RESTful interfaces that allow customers and partners to automatically manage translation activities with Acclaro, such as file transfers, order creation, quality ratings and status reporting. With the API in use, source content that resides in commercial or custom developed content management systems can be programmatically sent for translation, then subsequently received and published. Any questions or feedback regarding the Acclaro API can be directed to support@acclaro.com.

```

Comment:
What is the purpose of this file? There are a lot of overlaps with readme.

Reply:
this was requested by the ISV certification engineer